AncientReligionsRestored/common/religion/religions/00_celtic.txt

```diff
@@ -1,42 +1,42 @@
-﻿celtic_religion = {
+celtic_religion = {
 	family = rf_pagan
 	graphical_faith = pagan_gfx
-
+	
 	pagan_roots = yes
-
+	
 	doctrine = pagan_hostility_doctrine
-
+	
 	#Main Group
 	doctrine = doctrine_no_head
 	doctrine = doctrine_gender_male_dominated
 	doctrine = doctrine_pluralism_righteous
 	doctrine = doctrine_theocracy_temporal
-
+	
 	#Marriage
 	doctrine = doctrine_polygamy
 	doctrine = doctrine_divorce_allowed
 	doctrine = doctrine_bastardry_none
 	doctrine = doctrine_consanguinity_cousins
-
+	
 	#Crimes
 	doctrine = doctrine_homosexuality_shunned
-	doctrine = doctrine_adultery_men_accepted
-	doctrine = doctrine_adultery_women_accepted
+	doctrine = doctrine_adultery_men_shunned
```

Comment:
Did you change your mind on this? in our previous discussion it seemed like you thought this should be accepted.